lib/DataRepeater.js

```diff
@@ -215,9 +215,9 @@ var DataRepeater = module.exports = kind(
 	create: kind.inherit(function (sup) {
 		return function () {
 			sup.apply(this, arguments);
+			this.selectionType = this.multipleSelection ? (this.groupSelection ? 'group' : 'multi') : this.selectionType;
 			this.collectionChanged();
 			// Converting deprecated selection properties to our current selection API
-			this.selectionType = this.multipleSelection ? (this.groupSelection ? 'group' : 'multi') : this.selectionType;
```

Comment:
Because selectionType needs to be set before the collectionChanged method is invoked.